bundles/pom.xml

```diff
@@ -515,4 +516,4 @@
     </profile>
   </profiles>
 
-</project>
+</project>
```

Comment:
Can you leave the line feed?

Reply:
Ups, didn't recognized this after rebase.